docs/metrics.md

```diff
@@ -237,6 +237,9 @@ Histogram of VM phase transitions duration from deletion time in seconds. Type:
 ### kubevirt_vmi_phase_transition_time_seconds
 Histogram of VM phase transitions duration between different phases in seconds. Type: Histogram.
 
+### kubevirt_vmi_resource_requests
```

Comment:
@machadovilaca  Since this is a VM definition I would suggest to update the metric name to kubevirt_vm_resource_requests. WDYT?
We need to report it for non-running VMs as well.

Reply:
Please update

Reply:
I agree, once it's fixed I will lgtm (:

Reply:
this change no longer exists